src/mon/OSDMonitor.cc

```diff
@@ -1076,7 +1076,10 @@ void OSDMonitor::prime_pg_temp(
   const OSDMap& next,
   pg_t pgid)
 {
-  if (mon->monmap->get_required_features().contains_all(
+  if (mon->monmap->get_required_features().empty()) {
+    dout(10) << __func__ << ": quorum not formed" << dendl;
+    return;
+  } else if (mon->monmap->get_required_features().contains_all(
```

Comment:
maybe we don't need all of them. but i wanted to test it using rados qa suites sooner, so didn't look at it closer. but will do tomorrow.